action-allowedlist/approved.json

```diff
@@ -1,60 +1,87 @@
 [
     {
       "actionLink": "actions/checkout",
-      "actionVersion": "ec3a7ce113134d7a93b817d10a8272cb61118579"
+      "actionVersion": "a12a3943b4bdde767164f792f33f40b04645d846",
```

Comment:
ec3a7ce113134d7a93b817d10a8272cb61118579 was a duplicate that I deleted. Now the diff is messed, comparing the wrong entries everywhere.